### DISCUSSION TITLE
In-app tutorial.

### DISCUSSION THREAD

**Discussion by razvanab** [comfyanonymous/ComfyUI#4381] [Ideas]
I notice that the drag and add functionality has changed in the later update. People may not know this and grow frustrated at not knowing why it doesn't work anymore. So, I propose an in-app tutorial at first run to teach people the basics of using the UI.

**Comment by huchenlei** [comment]
Can you file a bug in https://github.com/Comfy-Org/ComfyUI_frontend about the unexpected behaviour?

**Reply by razvanab** [reply]
It's not a bug, I think. You just need to keep the shift key press when dragging a wire instead of just dragging the wire like in the legacy UI. But if you think this is not normal behavior, let me know and I will file a bug.
